src/App.js

```diff
@@ -42,11 +45,20 @@ function App() {
         }
     }
 
+    function startGame(gameHasStarted) {
```

Comment:
I think with the name "startGame" this should always start the game, then you don't need to pass a parameter as you're always calling setGameStarted(true)